### ISSUE TITLE
ion-datetime: the displayed time and the picker time are off by 1 hour

### ISSUE THREAD

**Issue by gigocabrera** [driftyco/ionic#9521]
**Ionic version:**  (check one with "x")
[ ] **1.x**
[x] **2.x**

**I'm submitting a ...**  (check one with "x")
[x] bug report
[ ] feature request
[ ] support request 

**Current behavior:**
Set the displayed time in ion-datetime to 12:01 AM and tap/click on the control. The picker time is showing 01:01 AM

**Expected behavior:**
If the displayed time in ion-datetime is 12:01 AM, the picker should also show 12:01 AM

**Steps to reproduce:**
This issue is reproducible using the demo from the Docs
1) Go to the [official ion-datetime page](http://ionicframework.com/docs/v2/api/components/datetime/DateTime/)
2) Select 12:01 AM for the Tokyo datetime picker under World Times (as shown below)
![ionic-datetime-issue01](https://cloud.githubusercontent.com/assets/6364855/20942253/9b8b851e-bbc8-11e6-81a6-e9711daa58cd.png)
3) Tap/click on the ion-datetime control again and you'll see the picker time is off by 1 hour
![ionic-datetime-issue02](https://cloud.githubusercontent.com/assets/6364855/20942322/ef2ea1b0-bbc8-11e6-8318-dccb85455d4b.png)

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

```
Your system information:

Cordova CLI: 6.4.0
Ionic Framework Version: 2.0.0-rc.3-201612021933
Ionic CLI Version: 2.1.13
Ionic App Lib Version: 2.1.7
Ionic App Scripts Version: 0.0.46
ios-deploy version: Not installed
ios-sim version: Not installed
OS: Windows 10
Node Version: v6.7.0
Xcode version: Not installed
```

**Comment by gigocabrera** [comment]
Any updates on this issue? Any workarounds?

**Comment by manucorporat** [comment]
Should be fixed in 2.3 release!
